boards/doc.txt

```diff
@@ -15,4 +15,56 @@
  * configuration of a controller and some external devices such as sensors or
  * radios. All aspects concerning configuration of GPIO pins, MCU clock and
  * device drivers should go into this module.
+ *
+
+Guide to board pinouts {#pinout_guide}
+=====================================
+
+The following are the steps to take to locate specific pinouts on boards.
+This is useful when connecting external hardware to RIOT supported boards.
+
+1. Check the board documentation to see if the pinout is available
```

Comment:
where to find that (as a newbie)? I.e. add a link to http://doc.riot-os.org/group__boards.html

Reply:
Done